BetterPasswordPlugin.inc.php

```diff
@@ -208,17 +199,109 @@ function checkPassword($hookName, $args) {
 			}
 		}
 	}
-
+        /**
+	 * Check for newly added blacklists. If ran the first time creates the list of blacklists
+	 * @return 0|1|2 0 if update failed, 1 if setting is updated, 2 if new setting has been created
```

Comment:
Anytime you want to introduce a magic number, introduce a constant instead.  The way we do this in PKP is a bit dated, but an example is here:
https://github.com/ulsdevteam/ojs-sushiLite-plugin/blob/5afdfa3eb4d8519f3a07fa6f0f264fd8521430e7/classes/SushiLite.inc.php#L15-L23

Note that these are introduced globally, rather than within the scope of the class (which would be a bit more modern), so they need to be very uniquely named (e.g. by starting the constant name with the name of the plugin).